### ISSUE TITLE
Verify contents and format of track documentation

### ISSUE THREAD

**Issue by kytrinyx** [exercism/xgroovy#35]
Each language track has documentation in the `docs/` directory, which gets included on the site
on each track-specific set of pages under [/languages](http://exercism.io/languages).

We've added some general guidelines about how we'd like the track to be documented in exercism/exercism.io#3315
which can be found at https://github.com/exercism/exercism.io/blob/master/docs/writing-track-documentation.md

Please take a moment to look through the documentation about documentation, and make sure that
the track is following these guidelines. Pay particularly close attention to **how to use images**
in the markdown files.

Lastly, if you find that the guidelines are confusing or missing important details, then a pull request
would be greatly appreciated.

**Comment by Dispader** [comment]
I don't know if it's helpful or not, but I gave a whirl at some edits to the `./docs` directory in Pull #37 .  As it says there, though— looks like all the files were there with no ultra-clear violations of the guidelines.

So, while I hope that folks think my little edits are improvements... even if they aren't, it's my little attempt to move the proofing of the docs forward.

Cheers!

**Comment by Dispader** [comment]
Okay, folks— with the merging of [Pull #37](https://github.com/exercism/xgroovy/pull/37), I'm thinking that we might be good here.  But I'd like to leave the issue open for some discussion or feedback for at least a few days, to give people an opportunity to comment, so I'm not going to close this just now.

**Comment by Dispader** [comment]
Hey, everyone.  I've been going over the issues again, and I'm still thinking this is close to done.  Not to drag feet, but I did another copy read and caught what I believe are a few errors, for which I've proposed some fixes over in Pull #39 .

I've put a call-out for an alternate copy read over in Twitter, and if we get a response, great!  Best case, maybe we get lucky and generate some more interest in a set of eyes on the Groovy track.  Worst case, we float this as open, merge Pull #39 up, and close this issue, methinks.
